docs/README.md

```diff
@@ -97,4 +97,14 @@ class Client
 		# get("resources", verify_peer: false)
 	end
 end
-```
\ No newline at end of file
+```
+
+### Using with `gitlab` gem
```

Comment:
I'm not interested in something specific to another gem living in the docs here. I think it would be better to have this in the gitlab gem docs or am I misunderstanding something?